internal/buffer/settings.go

```diff
@@ -7,13 +7,17 @@ import (
 	"github.com/zyedidia/micro/v2/internal/screen"
 )
 
-func (b *Buffer) SetOptionNative(option string, nativeValue interface{}) error {
+func (b *Buffer) SetOptionNative(option string, nativeValue interface{}, storeLocal bool) error {
```

Comment:
Breaks compatibility with plugins, right?

Reply:
At least `micro` internally there wasn't any usage of `SetOptionNative` in any plugins, but yes...since the interface is treated the same as `SetOption` for plugins...it breaks with them.

Again, one more round :sweat:

Reply:
FWIW I haven't reviewed the last round yet, I just noticed that it breaks my lua code.